chainer/serializers/npz.py

```diff
@@ -94,7 +94,9 @@ def __getitem__(self, key):
     def __call__(self, key, value):
         key = key.lstrip('/')
         dataset = self.npz[self.path + key]
-        if isinstance(value, numpy.ndarray):
+        if value is None:
```

Comment:
Do we need to fix document of `AbstractSerializer`? In its docstring, `value` is assumed to be `scalar, array, or str`.